src/corehost/cli/libhost.h

```diff
@@ -11,9 +11,14 @@
 enum host_mode_t
 {
     invalid = 0,
-    muxer,
-    standalone,
-    split_fx
+    
+    muxer,          // Invoked as "dotnet.exe".
+    
+    standalone,     // Invoked as "appname.exe" from the application base: either "standalone" or "branded". When implementing branded exes, rename this to "apphost"
+
+    split_fx        // Invoked as "corehost.exe" for xunit scenarios -- this has to be fixed by the CLI to not use this executable and this mode should not be supported.
```

Comment:
Is there an issue tracking this @piotrpMSFT @livarcocc? Also, what is the timeframe to fix this?